html/utilities.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Utilities</title>
+  <link rel="stylesheet" href="../css/main.css">
+</head>
+<body>
+  <h1>Utilities Module</h1>
+  
+  <section>
+    <h2>Flex Centered</h2>
+    <div class="flex-centered" style="background: gray; width: 30rem; height: 20rem;">
+      <h3>I'm centered</h3>
+    </div>
+  </section>
+
+</body>
+</html>
```

Comment:
Add line at end of file.

Reply:
Since the requested change is too minor, I have agreed with @esfox to change this. The line is added at https://github.com/esfox/aestheticss/pull/7/commits/19958769eb33c104c0e6f37ba99efc2add04c534 .